CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+All notable changes to this project will be documented in this file. If you make a notable change to the project, please add a line describing the change to the "unreleased" section. The maintainers will make an effort to keep the [Github Releases](https://github.com/IEA-Task-43/digital_wra_data_standard/releases) page up to date with this changelog. The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## [Unreleased]
+- feature 1
+- feature 2
+
+
+## [1.1] - YYYY-MM-DD
+- feature 1
```

Comment:
We decided to use semantic versioning (https://semver.org/) with the YYYY.MM tagged onto the end e.g. '0.1.0-2020.07'

Reply:
@Dynorat it looks like CHANGELOG.md still needs to be adjusted as pointed out by @stephenholleran before merging.

Reply:
roger that

Reply:
Looking good now. Brace for merging!